### ISSUE TITLE
Feature: In-game documentation

### ISSUE THREAD

**Issue by schindlershadow** [0999312/umapyoi#45]
### Describe the problem related to the feature request

no in-game documentation

### Describe the solution you'd like

in-game documentation or link to external documentation

### Describe alternatives you've considered

Chinese only wiki https://www.mcmod.cn/class/8546.html

**Comment by 0999312** [comment]
INSTALL PATCHOULI MOD

**Comment by 0999312** [comment]
OK, maybe the patchouli issue, Version and other info please.

**Comment by schindlershadow** [comment]
```
OS: linux
Launcher: neoforge 21.1.42
Java version: openjdk 21.0.4
Minecraft version: 1.21.1
Umapyoi version: umapyoi-1.0.2-1.21
patchouli 1.21-87-NEOFORGE
```
modlist https://pastebin.com/raw/ppvzu0Zj

**Comment by 0999312** [comment]
Fixed in 1.0.3, but sent book for new player need time for fixing, stay tuned.
